cmd/admxgen/defs/apparmor.yaml

```diff
@@ -16,3 +16,37 @@
   type: "apparmor"
   meta:
     strategy: append
+- key: "/apparmor-users"
+  displayname: "AppArmor"
+  explaintext: |
+    Define an AppArmor user profile to be parsed and loaded on client machines.
+    The profile is specified as a file path relative to the SYSVOL/apparmor/ubuntu/ directory.
+    On the client machine, user profiles are stored in /etc/apparmor.d/adsys/users/<user-name>, thus the administrator can reference abstractions and tunables shipped with the client distribution of AppArmor.
+
+    The profile will ideally contain a mapping between a user and a role. Roles must be configured beforehand in the System-wide application confinement section.
+    Below is an example of a user profile declaration:
+
+      ^{{ADSYS_USER}} {
```

Comment:
As this will be copied by us on disk and always stats with the same, should we, to avoid errors, only request to put the content of the template itself, and we encapsulate it with `user { … }`?

Reply:
Can we have some magic post-processing the file maybe? Like supporting {} maybe if the user wanted it? However, here, we really need to make something generic as you did with the templated name…